Source/WebCore/platform/mock/MockRealtimeVideoSource.h

```diff
@@ -83,6 +83,7 @@ class MockRealtimeVideoSource : public RealtimeVideoCaptureSource, private Orien
 
     void startProducingData() override;
     void stopProducingData() override;
+    void endProducingData() final;
```

Comment:
This is source of asserts here, I haven't found why...

Reply:
I'll revert that part for now.